app/components/preprint-status-banner/component.js

```diff
@@ -0,0 +1,99 @@
+import Ember from 'ember';
+
+const PENDING = 'pending';
+const ACCEPTED = 'accepted';
+const REJECTED = 'rejected';
+
+const PRE_MODERATION = 'pre-moderation';
+const POST_MODERATION = 'post-moderation';
+
+export const ICONS = Object.freeze(Ember.Object.create({
```

Comment:
Is there a reason for these to be `Ember.Object`s? They'll never change and nothing observes them, so they may as well be plain objects.